OpenOversight/app/models/config.py

```diff
@@ -10,6 +10,7 @@ class BaseConfig:
     def __init__(self):
         # App Settings
         self.DEBUG = False
+        self.ENV = os.environ.get("ENV")
```

Comment:
This variable is required or we end up failing a multitude of csv-related tests.

Reply:
Do you think it makes sense to have a default in that case then? Perhaps `testing` or `development`?

```suggestion
        self.ENV = os.environ.get("ENV", "development")
```